src/main/java/com/cloudant/client/api/CloudantClient.java

```diff
@@ -48,9 +48,7 @@
  * <pre>
  * CloudantClient client = new CloudantClient("mycloudantaccount","myusername","mypassword");
  * </pre>
- * <p/>

```

Comment:
In general a bit more formatting here would make the output a bit nicer - I know you don't have time to fix all of the doc formatting but this is an important class.